src/core/tsi/ssl_transport_security.cc

```diff
@@ -1662,9 +1662,9 @@ static tsi_result ssl_handshaker_do_handshake(tsi_ssl_handshaker* impl,
           const char* verify_err = X509_verify_cert_error_string(verify_result);
           verify_result_str = absl::StrCat(": ", verify_err);
         }
-        LOG(ERROR) << "Handshake failed with fatal error "
-                   << grpc_core::SslErrorString(ssl_result) << ": " << err_str
-                   << verify_result_str;
+        LOG(INFO) << "Handshake failed with fatal error "
```

Comment:
Wondering if we should downgrade the language here too - "Handshake failed with error" instead of "fatal error"?
WDYT?

Reply:
Great idea, done.